index.html

```diff
@@ -128,6 +128,10 @@ <h4 class="modal-title text-center">Welcome to Angry Coders</h4>
     $("#editor").css("display", "block");
     editor.setTheme("ace/theme/monokai");
     editor.getSession().setMode("ace/mode/javascript");
+    editor.setOptions({
+      fontFamily: "Fira Mono",
```

Comment:
need to import the font too. it is not present in everyone's computer right ? 

add 
```html
<link rel="stylesheet" href="https://code.cdn.mozilla.net/fonts/fira.css">
```